### ISSUE TITLE
How to train the model without depth

### ISSUE THREAD

**Issue by QingCheng24** [HarborYuan/PolyphonicFormer#9]
Hi @HarborYuan,

I'm trying to benchmark your model with our own dataset. As you mentioned in the paper, the first training step is only trained with the panoptic branch and data. 
I've set the `with_depth=False` from [this line](https://github.com/HarborYuan/PolyphonicFormer/blob/main/configs/_base_/models/polyphonic_former.py#L45) but there are errors about the data, loss, etc. Do I need to comment out all the depth-related lines manually?

Could you please give some instructions on how to do the panoptic branch pertaining?
Much appreciated

**Comment by HarborYuan** [comment]
Hi @QingCheng24 ,

Is the `find_unused_parameters` what you need? It may help you circumvent changing the code.

**Comment by QingCheng24** [comment]
Thank you for the quick reply @HarborYuan. 

It's about the variables in multiple forward() like `depth_feats`, `scaled_depth_preds`, `depth_proposal`, etc. and the depth loss. It seems with_depth=False from [this line](https://github.com/HarborYuan/PolyphonicFormer/blob/main/configs/_base_/models/polyphonic_former.py#L45) doesn't control the overall workflow. 

So I wonder whether there's some deep trick or I should manually refactor the code. Or to keep everything intact but not to include the depth loss is enough.

**Comment by HarborYuan** [comment]
Hi @QingCheng24 ,

I have a brief check about the code implementation. There was a problem when I was organizing the code for release. To quickly run the experiments without a depth branch, you can set the depth loss weight to 0 to train the model. The model will be identical to K-Net after that.
